test.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Cluster
```

Comment:
I'm guessing this wasn't supposed to be added. =)

Reply:
Yes sorry, I updated this in a later commit.